doc/spec/estimation/forest.rst

```diff
@@ -1,10 +1,156 @@
+can wrap any class that supports fit and predict and enables sample weight functionality. This is done either
+by re-sampling the data based on the weights and then calling fit and predict, or, in the case of linear losses,
+by re-scaling the features and labels appropriately based on the weights.
+
+
+    >>> est = ContinuousTreatmentOrthoForest(model_y=WeightedModelWrapper(Lasso(), sample_type=sample_type),
+    ...                                      model_t=WeightedModelWrapper(Lasso(), sample_type=sample_type))
+
+If the variable :code:`sample_type` takes the value "weighted", then the wrapper assumes the loss
+is the squared loss and the function class is linear and re-scales the features and labels appropriately.
+If not, then it re-samples the data based on the weights and calls the fit method of the base
+class on this re-sampled dataset. The latter has higher variance and should not be chosen if the
+first approach is applicable.
+
+In the case of discrete treatments (see :py:class:`~econml.ortho_forest.DiscreteTreatmentOrthoForest`) the
+method estimates :math:`\theta(x)` for some target :math:`x` by solving a slightly different
+set of equations (see [Oprescu2018]_ for a theoretical exposition of why a different set of
+estimating equations is used). In particular, suppose that the treatment :math:`T` takes
+values in :math:`\{0, 1, \ldots, k\}`, then to estimate the treatment effect :math:`\theta_t(x)` of
+treatment :math:`t` as compared to treatment :math:`0`, the method finds the solution to the
+equation:
+
+.. math::
+
+    \sum_{i=1}^n K(X_i, x)\cdot \left( Y_{i,t}^{DR} - Y_{i,0}^{DR}- \theta_t(x) \right) = 0
+
+where :math:`Y_{i,t}^{DR}` is a doubly robust based unbiased estimate of the counterfactual
+outcome of sample :math:`i` had we treated it with treatment :math:`t`, i.e.:
+
+.. math::
+    
+    Y_{i,t}^{DR} = \hat{\E}[Y \mid T=t, x, W_i] - 1\{T_i=t\} \frac{Y_i - \hat{\E}[Y \mid T=t, x, W_i]}{\hat{\E}[1\{T=t\} \mid x, W_i]} 
+
+where :math:`\hat{\E}[Y \mid T=t, x, W_i]` and :math:`\hat{\E}[1\{T=t\} \mid x, W_i]` are first stage estimates of the
+corresponding conditional expectations. These two regression functions are fitted in a similar manner
+as in the continuous treatment case. However, in the case of discrete treatment, the model for the treatment is 
+a multi-class classification model and should support :code:`predict_proba`.    
+
+For more details on the input parameters of the orthogonal forest classes and how to customize
+the estimator checkout the two modules:
+    - :py:class:`~econml.ortho_forest.DiscreteTreatmentOrthoForest`
+    - :py:class:`~econml.ortho_forest.ContinuousTreatmentOrthoForest`
+
+
+For more examples check out our 
+`OrthoForest Jupyter notebook <https://github.com/Microsoft/EconML/blob/master/notebooks/Orthogonal%20Random%20Forest%20Examples.ipynb>`_ 
+
+Examples
+--------
```

Comment:
Consider making this a sub-subheading by using a different underline character, such as `^^^^^^^`